src/mender-inventory-mender-configure

```diff
@@ -60,7 +60,10 @@ if [ -f "$CONFIG_CHECKSUM" ] && [ "$COMPUTED_CHECKSUM" = "$(cat "$CONFIG_CHECKSU
     exit 0
 fi
 
+CAPTURED_CURL_OUTPUT=$(mktemp)
 # Do the report.
+set +e
+{
```

Comment:
Not really necessary, but I thought it look'd nice. Say aye and I will go the traditional route.